README.md

```diff
@@ -49,6 +49,8 @@ This microservice allows sending and receiving messages via FIX protocol
 + *useNextExpectedSeqNum* - session management based on next expected sequence number. (`false` by default)
 + *saveAdminMessages* - defines if admin messages will be saved to internal outgoing buffer. (`false` by default)
 + *resetStateOnServerReset* - whether to reset the server sequence after receiving logout with text `Next Expected MSN too high, MSN to be sent is x but received y`.
++ **connectionTimeoutOnSend** - timeout in milliseconds for sending message from queue thread. _Default, 30000 mls._
+  If connection is not established within the specified timeout an error will be reported.
```

Comment:
Please add the https://www.rabbitmq.com/consumers.html#acknowledgement-timeout link to README